docs/user/_sidebar.md

```diff
@@ -1,7 +1,8 @@
-- [Kyma Telemetry Module](README.md)
-- [Telemetry Manager](01-manager.md)
-- [Application Logs](02-logs.md)
-- [Traces](03-traces.md)
-- [Metrics](04-metrics.md)
-- [Tutorials](05-tutorials.md)
-- [Resources](resources/README.md)
\ No newline at end of file
+- [Home](https://kyma-project.io/#/)
```

Comment:
Seeing that one it does not have the intendation as you propose: https://raw.githubusercontent.com/kyma-project/serverless-manager/main/docs/user/_sidebar.md

Reply:
I also think we should use [Home] (/_) _(space added on purpose)_. Unless there's been a change I'm not aware of.

Reply:
I removed the indentation and changed the link to a relative one. Thank you both for spotting this!